Homework4: Andrei Bartosh

sharp-0000ff opened the pull request.

sharp-0000ff committed:
release 0.4

belovmd reviewed (changes requested):
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-python-spring/blob/master/homework_review/Homework4.md):
- 1
- 8
- 31
- 21
- 13
- 20

sharp-0000ff committed:
Make fixes after Code Review
Task 2: changed complexity algorithm from O(n**2) to O(n) and renamed variables
Task 3, 4: changed solution
Task 6: changed to one-line solution
Task 7: return value changed

belovmd commented on src/second_project.py:
```diff
@@ -0,0 +1,16 @@
+"""Дан список стран и городов каждой страны. Затем даны названия городов.
```
doesn't handle situation with Brest in Belarus and France

belovmd commented on src/sixth_project.py:
```diff
@@ -0,0 +1,12 @@
+"""
+
+words = "My name   is Andrei  name  Sarha Thank  Andrei you\nfriend \tOh La La"
+print(len(words.split()))
```
it will send all words, need unique words. task not solved.

belovmd reviewed (commented):
review p.8 not addressed
7,5
